api/OpenAI.net8.0.cs

```diff
@@ -6815,4 +6817,31 @@ public enum VectorStoreStatus {
         Completed = 2,
         Expired = 3
     }
+}
+namespace OpenAI.Videos {
+    [Experimental("OPENAI001")]
+    public class VideoClient {
+        protected VideoClient();
+        public VideoClient(ApiKeyCredential credential, OpenAIClientOptions options);
+        public VideoClient(ApiKeyCredential credential);
+        public VideoClient(AuthenticationPolicy authenticationPolicy, OpenAIClientOptions options);
+        public VideoClient(AuthenticationPolicy authenticationPolicy);
+        protected internal VideoClient(ClientPipeline pipeline, OpenAIClientOptions options);
+        public VideoClient(string apiKey);
+        [Experimental("OPENAI001")]
+        public Uri Endpoint { get; }
+        public ClientPipeline Pipeline { get; }
+        public virtual ClientResult CreateVideo(BinaryContent content, string contentType, RequestOptions options = null);
+        public virtual Task<ClientResult> CreateVideoAsync(BinaryContent content, string contentType, RequestOptions options = null);
+        public virtual ClientResult CreateVideoRemix(string videoId, BinaryContent content, string contentType, RequestOptions options = null);
+        public virtual Task<ClientResult> CreateVideoRemixAsync(string videoId, BinaryContent content, string contentType, RequestOptions options = null);
+        public virtual ClientResult DeleteVideo(string videoId, RequestOptions options = null);
+        public virtual Task<ClientResult> DeleteVideoAsync(string videoId, RequestOptions options = null);
+        public virtual ClientResult GetVideo(string videoId, RequestOptions options = null);
+        public virtual Task<ClientResult> GetVideoAsync(string videoId, RequestOptions options = null);
+        public virtual ClientResult GetVideos(long? limit = null, string order = null, string after = null, RequestOptions options = null);
+        public virtual Task<ClientResult> GetVideosAsync(long? limit = null, string order = null, string after = null, RequestOptions options = null);
+        public virtual ClientResult RetrieveVideoContent(string videoId, string variant = null, RequestOptions options = null);
+        public virtual Task<ClientResult> RetrieveVideoContentAsync(string videoId, string variant = null, RequestOptions options = null);
```

Comment:
Rename these methods to "DownloadVideo" and "DownloadVideoAsync" for consistency with other APIs (such as "DownloadFile").